ui-tests/src/test/java/io/syndesis/qe/pages/integrations/CiCd/ManageCICDPage.java

```diff
@@ -0,0 +1,66 @@
+package io.syndesis.qe.pages.integrations.CiCd;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.List;
+
+public class ManageCICDPage extends SyndesisPageObject {
+
+    private static final class Element {
+        public static final By ROOT = By.className("pf-c-page__main");
+        public static final By ITEM_LIST_TAGS = By.className("list-group-item");
+        public static final By TAG_NAME = By.className("list-group-item-heading");
+    }
+
+    private static final class Button {
+        public static final By ADD_NEW_EMPTY_LIST = By.cssSelector("[data-testid=\"cicd-list-empty-state-add-new-button\"]");
+        public static final By ADD_NEW = By.cssSelector("[data-testid=\"cicd-list-view-add-new-button\"]");
+        public static final By EDIT = By.cssSelector("[data-testid=\"cicd-list-item-create-button\"]");
+        public static final By REMOVE = By.cssSelector("[data-testid=\"cicd-list-item-remove-button\"]");
+    }
+
+    @Override
+    public SelenideElement getRootElement() {
+        return $(ManageCICDPage.Element.ROOT).shouldBe(visible);
+    }
+
+    @Override
+    public boolean validate() {
+        return getRootElement().is(visible);
+    }
+
+    public void clickOnAddNewTagButton() {
+        if (this.getAllTags().isEmpty()) {
```

Comment:
Did you consider writing a css selector that would match both and get rid of this if? Such as `"[data-testid=\"cicd-list-empty-state-add-new-button\"],data-testid=\"cicd-list-view-add-new-button\"]"`

Reply:
I haven't known that it works that way, thanks.